new/LIT7063Devils.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7063Devils" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አላሁማ፡ ወያኑራ፡ ሐሽም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAlahuma wa-yānurā ḥašǝm</title>
+                <title xml:lang="en" corresp="#t1">ʾAlahuma and Yānurā ḥašǝm</title>
```

Comment:
I am not sure that this translation makes sense

At the same time, this is obviously a sequence of Arabic/Islamic names/terms used for asmat, which could be interesting if one looks for Muslim/Arabic references, so worth marking up

Reply:
What is your suggestion? Do you want another translation or should I not translate it and delete the translation in this record?

Reply:
I would delete the translation and rather add some keyword/description to make it searchable if one looks for Islamic allusions, and Arabic loans; at the moment these aspects are obscured and not retrievable

Reply:
possibly this or similar prayer is among those which were studied by Griaule, he published on Islamic formulae used in Christian prayers, could be checked (_Le livre de recettes d'un dabtara abyssin_). Also Strelcyn published separately on the phenomenon.

Reply:
> possibly this or similar prayer is among those which were studied by Griaule, he published on Islamic formulae used in Christian prayers, could be checked (_Le livre de recettes d'un dabtara abyssin_). Also Strelcyn published separately on the phenomenon.

OK. I will check that as soon as possible.

Reply:
not yet updated?

Reply:
No, I missed to have a look to the HLCEE library, when I was in Hamburg. I will possibly find Griaule in Marburg next week or I will do it in two weeks, when I am back in Hamburg.